var/spack/repos/builtin/packages/cgns/package.py

```diff
@@ -48,6 +49,11 @@ def cmake_args(self):
                 '-DCMAKE_Fortran_COMPILER=%s' % spec['mpi'].mpifc
             ])
 
+        if '+int64' in spec:
+            options.extend([
+                '-DCGNS_ENABLE_64BIT:BOOL=ON'
+            ])
```

Comment:
Can you rewrite this as:
```python
options.append(
    '-DCGNS_ENABLE_64BIT:BOOL={0}'.format('ON' if '+int64' in spec else 'OFF')
)
```
i.e. forcing the option to be `OFF` if `'~int64' in spec`?

Reply:
Done.